unity/Holo/Assets/Scripts/STLConverter.cs

```diff
@@ -0,0 +1,178 @@
+﻿using Unity;
+using UnityEngine;
+using UnityEditor;
+using System.Text;
+using System.Collections.Generic;
+using System.IO;
+
+/// A class for importing and converting StL series into an animation
+public class STLSeriesConverter
+{
+    GameObject series_GameObject;
+    
+    [MenuItem("Holo/Convert STL series to a .prefab")]
+    public static void ConvertSTL()
+    {
+        STLSeriesConverter stlSeriesConverter = new STLSeriesConverter();
+        STLSeriesImporter stlImporter = new STLSeriesImporter();
+
+        stlSeriesConverter.series_GameObject = stlImporter.Get_GameObject();
+        stlSeriesConverter.series_GameObject.AddComponent<BlendShapeAnimation>();
+
+        stlSeriesConverter.ExportToPrefab();
+    }
+
+    // Exports finished GameObject to a .prefab
+    private void ExportToPrefab()
+    {
+        //string save_path = EditorUtility.SaveFilePanel("Choose a folder where for a .prefab file", "", "", ".prefab");
+        string save_path = @"C: \Users\mit - kuchnowski\repos\holo\unity\Holo\Assets\Resources\test2.prefab";
+        PrefabUtility.SaveAsPrefabAsset(series_GameObject, save_path);
+    }
+
+}
+
+//A class for importing a ASTL series from a dir
+public class STLSeriesImporter
+{
+    GameObject imported_STLs = new GameObject("Hypertrophy");
+
+    private string[] file_paths;
+
+    public GameObject Get_GameObject()
+    {
+        return this.imported_STLs;
+    }
+
+    // Object constructor, initiates STL series import.
+    public STLSeriesImporter()
+    {
+        this.Get_file_paths();
+        this.Load_files();
+    }
+
+
+    // gets path to the subsequent STL meshes stored in a root folder.
+    private void Get_file_paths()
+    {
+        //string root_folder = EditorUtility.OpenFolderPanel("Select STL series root folder", "", "");
+        string root_folder = @"C:\Users\mit-kuchnowski\repos\holo\data\hypertrophy-heart\meshes\";
+        file_paths = Directory.GetFiles(root_folder);
+    }
```

Comment:
This can also be remade to use a relative path, inside project's Assets/, so it works on everyone's machines.

`Directory.GetFiles` is not native to Unity (it's from C# System.IO), but you can prepend the `Application.dataPath` from Unity ( https://docs.unity3d.com/ScriptReference/Application-dataPath.html ). So something like

```
... = Directory.GetFiles(Application.dataPath + "\hypertrophy-heart\meshes\")
```

may work.